mqttclients/ts/mqttjsClientExtensions/src/geoJsonPoint.ts

```diff
@@ -0,0 +1,9 @@
+export class GeoJsonPoint {
```

Comment:
should we use https://github.com/caseycesari/geojson.js/ ?

Reply:
Not sure. That one is 6-years old. I was keeping it simple, but could go either way.

Reply:
GeoJSON has not changed in 6 years, sometimes is not a bad thing, just means it's _DONE_.

maybe there is a more recent library, this is just to show how to adapt to different messaging formats/patterns.